static/css/basic.css

```diff
@@ -0,0 +1,41 @@
+* {
+    font-family:Verdana, Geneva, Tahoma, sans-serif;
+}
+
+body {
+    margin-left:200px;
+    margin-right:200px;
```

Comment:
for large-value settings, typical to use either a % of container element or vw / vh